### PR TITLE
Remove the dead code and CSS issue #180 #116

### PR DESCRIPTION
**Pull request by RahulBansal0502**
I have removed the dead code from the repo with reference to issue #180 #116 . Please check @ArunTeltia @LordShenron @yasharth291

### CODE REVIEW

**Review by yasharth291 — APPROVED** [review]

**Comment by yasharth291** [comment]
LGTM

**Comment by ArunTeltia** [comment]
How this is dead code ? @RahulBansal0502 , is this not bieng used in any of the html files ?

**Comment by RahulBansal0502** [comment]
No

**Comment by ArunTeltia** [comment]
Oh is it so should not t be better to first check if some file using A file or B file and if we delete B file and some file is using B file it will completely break the html page, can you check for it @RahulBansal0502

**Comment by RahulBansal0502** [comment]
I have checked @ArunTeltia and website is working smoothly. I found no issue and if you found any issue please let me know.

**Comment by ArunTeltia** [comment]
Okay give me some time @RahulBansal0502 I will have a look by myself in my spare time :)

**Comment by RahulBansal0502** [comment]
Ok

**Comment by ArunTeltia** [comment]
@RahulBansal0502 can you check your discord

File: css/style.css [hunk 1]
```diff
@@ -49,65 +49,6 @@ body {
   margin: 0 auto;
 }
 
-/*---------- FORMS ----------*/
-
-input[type="text"],
```
Comment (ArunTeltia) [inline css/style.css:54]: why are these deleted @RahulBansal0502

Comment (ArunTeltia) [reply css/style.css:54]: why are these changes made in the file 
Sorry got wrapped up with some work

**Review by ArunTeltia — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (RahulBansal0502) [reply css/style.css:54]: Actually in our website we are not using any type of form where user will have to enter anything so there is no need of forms.

Comment (ArunTeltia) [reply css/style.css:54]: okay, makes sense i thought they were unintentional will review this as well in some time, I confirmed that css file is indeed a dead code, great work noticing that

Comment (RahulBansal0502) [reply css/style.css:54]: Ya sure thanks!

**Comment by ArunTeltia** [comment]
so we are working on new index page here is the beta version https://projectsakura.xyz/index_rebase.html
so i dont think we need to think about this page 
so I dont think i need to review it add every file that are only used in index file and delete it or 

Better you can remove these chanes and can create a new pr for the other file @RahulBansal0502
